### PR TITLE
Replace figaro with rails credentials

### PR DESCRIPTION
**Pull request by chalchuck**
#### What does this PR do?

Replace the figaro gem with Rails Credentials (Default in Rails)

#### Description of Task to be completed

To use an ENV variable,
make a call to `Rails.application.credentials.env_variable_name` instead of `ENV['env_variable_name']`

If you need to add extra variables; 

for production environment, 
ensure you have the correct master.key @kevinsamoei please add the key to the Slack channel
run `bin/rails credentials:edit` from your terminal, edit using YAML syntax and Ctl+X to save and exit

for development/staging/test,
open %{test development staging}.yml and the credential.

PS: all credentials are defined and accessed using lower case alphanumerics

### CODE REVIEW

File: config/creds/development.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+bp_andela_media_database:           'bp-andela-media'
```
Comment (kevinsamoei) [inline config/creds/development.yml]: @chalchuck Aren't these supposed to be hidden. What happens when it is production?

Comment (chalchuck) [reply config/creds/development.yml]: the production ENV variables are in config/credentials.yml.enc, master.key is used to decrypt them when accessed by the app. @kevinsamoei

File: config/creds/test.yml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+bp_andela_media_database:           'bp-andela-media'
+bp_andela_media_database_password:  'databasepassword'
+bp_andela_media_database_username:  'postgres'
+bp_andela_media_database_port:      '5432'
+bp_andela_media_test_database:      'bp-andela-media-test'
+bp_andela_media_database_password:  'localhost'
```
Comment (gr1d99) [inline config/creds/test.yml]: `bp_andela_media_database_password` is duplicated.

**Comment by gr1d99** [comment]
since we are doing away with `ActiveCable` I also think we should remove references to its associated files and directories such as  `app/channels/*`, `app/assets/javascripts/channels/` directories and `app/assets/javascripts/cable.js` file.

**Comment by chalchuck** [comment]
Sure, we can remove all references to whatever part of Rails we are not using at the moment.

**Comment by chalchuck** [comment]
@gr1d99 I have removed the references to `ActionCable`

**Pull request merged**